kod/object/item/passitem/defmod.kod

```diff
@@ -140,6 +140,18 @@ messages:
    % This will reduce the damage that armor is taking in battle currently.
    DefendingHit(who = $,what = $)
    {
+      if poOwner <> $
+         AND IsClass(poOwner,&Player)
```

Comment:
I considered adding this check already for arenas (using the NoReagents message) but sometimes this will be the only place people use their special gear, thus it should probably degrade there to promote turnover of these items. We don't have a lot of special items yet, but if/when we add them it could be an issue.